Core/include/Acts/Vertexing/HelicalTrackLinearizer.ipp

```diff
@@ -76,7 +76,8 @@ Acts::Result<Acts::LinearizedTrack> Acts::
   ActsScalar qOvP = paramsAtPCA(BoundIndices::eBoundQOverP);
 
   ActsScalar m0 = params.particleHypothesis().mass();
-  ActsScalar p = std::abs(params.particleHypothesis().absoluteCharge() / qOvP);
+  ActsScalar absoluteCharge = params.particleHypothesis().absoluteCharge();
+  ActsScalar p = std::abs(absoluteCharge / qOvP);
```

Comment:
p can also be obtained by the particle hypothesis which is preferable in general IMO